src/pages/achievements/[id].tsx

```
@@ -96,14 +99,38 @@ const Achievement = () => {
 
+  const mintable = useMemo(() => {
+    return achievement?.community?.can_mint_certificates || isICPSubmission;
+  }, [achievement, isICPSubmission]);
+
   const isNotCertificateIcon = useMemo(() => {
     return !achievement?.metadata?.image?.includes("/img/certificates/");
   }, [achievement]);
 
+  const addCourseCompletionToTheIssuerCanister = useCallback(async () => {
+    if (!achievement?.metadata) return;
+    const { name } = achievement?.metadata;
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)